jax/_src/pjit.py

```diff
@@ -991,6 +991,16 @@ def _pjit_jaxpr(fun, out_shardings_thunk, out_layouts_thunk, in_type, debug_info
   canonicalized_out_shardings_flat, out_layouts_flat = _check_and_canonicalize_out_shardings(
       out_shardings_thunk, out_layouts_thunk, out_tree, tuple(out_type),
       jaxpr.jaxpr.debug_info, device_or_backend_set)
+
+  if config.enable_key_reuse_checks.value:
+    try:
+      from jax.experimental.key_reuse._core import check_key_reuse_jaxpr # pytype: disable=import-error
```

Comment:
Can you put this import at the top please?

Reply:
Nope, it leads to a circular import that is not easily detangled, because ~everything depends on pjit.